pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -145,67 +138,71 @@ func (p *Plugin) Configure(ctx context.Context, req *configv1.ConfigureRequest)
 		setDefaultValues(&config.Clusters[i])
 	}
 
-	if err = p.setConfig(config); err != nil {
-		return nil, status.Errorf(codes.Internal, "unable to set configuration: %v", err)
+	clients, err := p.hooks.newKubeClients(config)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "unable to create new kubeClients: %v", err)
+	}
+
+	stopCh := make(chan struct{})
```

Comment:
Should we close this if startInformers fails?

Reply:
good catch, updated with this change